### ISSUE TITLE
async/await feature

### ISSUE THREAD

**Issue by itsmrTech** [pusherman/react-native-network-info#38]
Hi.
is there anyway to use await for getting network info? I can't use callback.

**Comment by jamesjara** [comment]
Easy, just wrap it
```
function _getSSID(){
   return new Promise((resolve, reject)=>{
      NetworkInfo.getSSID(ssid => resolve(ssid))
   })
}
async getSSID(){
   cost SSID =  await _getSSID()
   return SSID
}
```

**Comment by pusherman** [comment]
Available in 5.0.0
